inc/fields/actorfield.class.php

```diff
@@ -248,6 +248,11 @@ public function prepareQuestionInputForSave($input) {
 
    public function parseAnswerValues($input) {
       $key = 'formcreator_field_' . $this->fields['id'];
+      if (!isset($input[$key])) {
+         $this->value = [];
+         return true;
+      }
+
```

Comment:
unrelated issue with Actors field: if the field is submitted empty, then there is no value in $_POST (instead of the previously expected empty array)